data/language/en-GB.txt

```diff
@@ -3822,3 +3822,4 @@ STR_6774    :Time since last inspection
 STR_6775    :{COMMA16} minute
 STR_6776    :{COMMA16} minutes
 STR_6777    :more than 4 hours
+STR_6778    :Rebuilding scenario index
```

Comment:
I don't think we need a new string for this, tbh. Instead, just reuse `STR_CHECKING_SCENARIO_FILES`.